### PR TITLE
Add new bts codes

### PR DESCRIPTION
**Pull request by AntonellaTatarelli**
Adding probes to acquire electromyographic activity of the left leg

### CODE REVIEW

File: devices/FreeEmgWearableDevice/applications/FREEEMG_Log.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
     <module>
         <name>yarprobotinterface</name>
-        <parameters>--config freeemg_logger.xml --loggerType "(matlab)" --period 0.01 --experiment-name free_emg_data --experiment-dir C:/tmp/ifeel_data/ </parameters>
+        <parameters>--config freeemg_logger.xml --loggerType "(matlab)" --period 0.01 --experiment-name free_emg_data --path C:/tmp/ifeel_data/ </parameters>
```
Comment (RiccardoGrieco) [inline devices/FreeEmgWearableDevice/applications/FREEEMG_Log.xml]:
```suggestion
        <parameters>--config freeemg_logger.xml --loggerType "(matlab)" --period 0.01 --experiment-name free_emg_data --path C:/tmp/data/ </parameters>
```
Probably it makes sense to change the **default** directory path this way 👀 .

cc @lrapetti

File: devices/FreeEmgWearableDevice/conf/freeemg_analog.xml [hunk 2]
```diff
@@ -76,6 +76,18 @@
...
+    <device type="iwear_wrapper" name="FreeEmgWearableDeviceWrapper">
+        <param extern-name="period" name="period">0.005</param><!--Device update period in seconds-->
+        <param extern-name="dataPortName" name="dataPortName">/FreeEmg/WearableData/data:o</param>
+        <action phase="startup" level="5" type="attach">
+            <paramlist name="networks">
+                <elem name="FreeEmgWearableDeviceWrapperLabel"> FreeEmgWearableDevice </elem>
+            </paramlist>
+        </action>
+        <action phase="shutdown" level="4" type="detach"/>
+    </device>
```
Comment (RiccardoGrieco) [inline devices/FreeEmgWearableDevice/conf/freeemg_analog.xml:89]: This is already in a dedicated configuration file: https://github.com/ami-iit/yarp-devices-bts-freeemg/blob/c8d3ed36c64995d481883f9feb1b5305f19f12d2/devices/FreeEmgWearableDevice/conf/freeemg.xml#L80-L88

Are you sure you want to put this device also here?

Comment (lrapetti) [reply devices/FreeEmgWearableDevice/conf/freeemg_analog.xml:89]: I actually don't see the wrapper in any of the other files in [`yarp-devices-bts-freeemg/devices/FreeEmgWearableDevice/conf`](https://github.com/ami-iit/yarp-devices-bts-freeemg/tree/main/devices/FreeEmgWearableDevice/conf). `freeemg_analog` streams the values as analogue data using the [`iwear_logger`](https://github.com/ami-iit/yarp-devices-bts-freeemg/blob/c8d3ed36c64995d481883f9feb1b5305f19f12d2/devices/FreeEmgWearableDevice/conf/freeemg_analog.xml#L80-L113) device, not the wearable data. Is there any other application streaming the wearable data?

Comment (RiccardoGrieco) [reply devices/FreeEmgWearableDevice/conf/freeemg_analog.xml:89]: > I actually don't see the wrapper in any of the other files in [`yarp-devices-bts-freeemg/devices/FreeEmgWearableDevice/conf`](https://github.com/ami-iit/yarp-devices-bts-freeemg/tree/main/devices/FreeEmgWearableDevice/conf). `freeemg_analog` streams the values as analogue data using the [`iwear_logger`](https://github.com/ami-iit/yarp-devices-bts-freeemg/blob/c8d3ed36c64995d481883f9feb1b5305f19f12d2/devices/FreeEmgWearableDevice/conf/freeemg_analog.xml#L80-L113) device, not the wearable data. Is there any other application streaming the wearable data?

Data is streamed as wearable data in `freeemg.xml` as mentioned in my [previous comment](https://github.com/ami-iit/yarp-devices-bts-freeemg/pull/6#discussion_r1205804899), and that is actually the purpose of that file.
With this PR data is also streamed as wearable message in `freeemg_analog.xml`, which will then stream the data in both ways (analog, wearable) at the same time. Is this the intended behaviour?

Comment (lrapetti) [reply devices/FreeEmgWearableDevice/conf/freeemg_analog.xml:89]: > Data is streamed as wearable data in `freeemg.xml` as mentioned in my [previous comment](https://github.com/ami-iit/yarp-devices-bts-freeemg/pull/6#discussion_r1205804899), and that is actually the purpose of that file.

Sorry, I missed that.

Then I agree there is no need to add it to this file. @AntonellaTatarelli I think you can remove the `iwear_wrapper` from the `freeemg_analog.xml` file

**Pull request merged**